client/src/components/AppFrame/AppFrame.js

```diff
@@ -10,7 +10,8 @@ import Typography from '@material-ui/core/Typography'
 import Divider from '@material-ui/core/Divider'
 import IconButton from '@material-ui/core/IconButton'
 import MenuIcon from '@material-ui/icons/Menu'
-//
+
+import Infinite from 'react-infinite'
```

Comment:
do you need to commit this in the `package.json` and `package-lock.json`?

Reply:
yup, sorry. Temporary moment of insanity